### ISSUE TITLE
PHP 8.1(.3) support

### ISSUE THREAD

**Issue by DanielRuf** [aaemnnosttv/wp-sqlite-db#27]
I am getting this:

```
Deprecated: Return type of WP_SQLite_DB\PDOEngine::beginTransaction() should either be compatible with PDO::beginTransaction(): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /Users/danielruf/projects/run-local-wordpress/wp-content/db.php on line 2429
```

**Comment by ssnepenthe** [comment]
And these:

```
Deprecated: Return type of WP_SQLite_DB\PDOEngine::commit() should either be compatible with PDO::commit(): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /home/me/Code/wp-sqlite-db/src/db.php on line 2445
Deprecated: Return type of WP_SQLite_DB\PDOEngine::query($statement, $mode = PDO::ATTR_DEFAULT_FETCH_MODE, ...$fetch_mode_args) should either be compatible with PDO::query(string $query, ?int $fetchMode = null, mixed ...$fetchModeArgs): PDOStatement|false, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /home/me/Code/wp-sqlite-db/src/db.php on line 1324
Deprecated: Return type of WP_SQLite_DB\PDOEngine::rollBack() should either be compatible with PDO::rollBack(): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /home/me/Code/wp-sqlite-db/src/db.php on line 2456
```

**Comment by ecotechie** [comment]
Same issue here, what's the latest PHP version supported?

**Comment by aaemnnosttv** [comment]
This should be fixed as of 1.3.0 👍
